docs/api/glossary.rst

```diff
@@ -0,0 +1,80 @@
+.. SPDX-License-Identifier: MIT
+
+.. currentmodule:: disnake
+
+Glossary
```

Comment:
not entirely sure about this, maybe it should go in api/index.rst after the toc. Will investigate further